keyboards/keyhive/maypad/keymaps/via/keymap.c

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2019 codybender
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+LAYOUT_ortho_5x4(
+    KC_A, KC_B, KC_C, KC_D,
+    KC_E, KC_F, KC_G, KC_H,
+    KC_I, KC_J, KC_K, KC_L,
+    KC_M, KC_N, KC_O, KC_P,
+    KC_Q, KC_R, KC_S, KC_T
+)
+
+};
+
+
+void matrix_init_user(void) {
+
+}
+
+void matrix_scan_user(void) {
+
+}
+
+void led_set_user(uint8_t usb_led) {
+
+}
```

Comment:
```suggestion
    LAYOUT_ortho_5x4(
        KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
        KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
        KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
        KC_P1,   KC_P2,   KC_P3,   KC_PENT,
        KC_P0,   KC_P0,   KC_PDOT, KC_PENT
    )
};
```
Generally the default VIA keymap should look as close as possible to the default keymap.